### PR TITLE
fixed bags with access to profile

### PR DESCRIPTION
**Pull request by andrey11m**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/controller/ProfileController.java [hunk 1]
```diff
@@ -4,24 +4,26 @@
...
 @RestController
 @RequiredArgsConstructor
 public class ProfileController {
 
     private final ProfileService profileService;
 
-    @GetMapping("/{id}")
-    public ProfileDTO getProfileById(@PathVariable Long id) {
-        return profileService.getById(id);
+    @GetMapping("")
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/ProfileController.java:19]: ("") is redundant, use just @GetMapping

File: src/main/java/internship/mbicycle/storify/controller/ProfileController.java [hunk 2]
```diff
@@ -4,24 +4,26 @@
...
+    public ProfileDTO getProfile(@ApiIgnore Principal principal) {
+        return profileService.getByEmail(principal.getName());
     }
 
-    @PutMapping("/{id}")
-    public ProfileDTO updateProfile(@PathVariable Long id, @Valid @RequestBody ProfileDTO profileDTO) {
-        return profileService.updateProfile(id, profileDTO);
+    @PutMapping("")
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/ProfileController.java:24]: ("") is redundant

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/controller/ProfileController.java (hunk 1 above)
Comment (mariyazaiceva) [reply src/main/java/internship/mbicycle/storify/controller/ProfileController.java]: Done

File: src/main/java/internship/mbicycle/storify/controller/ProfileController.java (hunk 2 above)
Comment (mariyazaiceva) [reply src/main/java/internship/mbicycle/storify/controller/ProfileController.java]: Done

**Pull request merged**